components/delegation/DelegationCenterMenu.tsx

```diff
@@ -19,12 +19,14 @@ import NewSubDelegationComponent from "./NewSubDelegation";
 import DelegationHTML from "./html/DelegationHTML";
 import WalletCheckerComponent from "./walletChecker/WalletChecker";
 import { useRouter } from "next/router";
+import NewAssignPrimaryAddress from "./NewAssignPrimaryAddress";
 
 export enum DelegationCenterSection {
   CENTER = "delegation-center",
   REGISTER_DELEGATION = "register-delegation",
   REGISTER_SUB_DELEGATION = "register-sub-delegation",
   REGISTER_CONSOLIDATION = "register-consolidation",
+  ASSING_PRIMARY_ADDRESS = "assign-primary-address",
```

Comment:
@prxt6529 should this be `ASSIGN_PRIMARY_ADDRESS`?

Reply:
sure should be!

Reply:
updated